esb/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/refactoring/delete/ESBMetaDataFileDeleteChange.java

```diff
@@ -91,9 +101,17 @@ private void identifyReplaces() throws IOException {
 
 			if (isArtifacts) {
 				isArtifactLine=false;
-				if (!isArtifact && line.trim().startsWith(artifactStart)) {
+				if (!isArtifact && line.trim().contains(artifactStart)) {
 					int artifactTagIndex = line.indexOf(artifactStart);
-					startIndex = fullIndex + artifactTagIndex;
+					spaceCount =0;
+					for (int stringIndex = artifactTagIndex-1; stringIndex >= 0 ; stringIndex--) {
+						if(line.charAt(stringIndex)==' '){
+							spaceCount++;
+						}else{
```

Comment:
Seems formatting not properly done, please fix it. Check other line of codes for formatting